### PR TITLE
Feature/changesets on work packages

### PR DESCRIPTION
**Pull request by ulferts**
Allows changesets to be associated to work_packages instead of issues.

Doing this enables exporting a work_package as pdf.

Additionally it was easy to let textile links (#..) to a work_package be handled equally.

OP issue: https://www.openproject.org/issues/1493

### CODE REVIEW

**Comment by mfrister** [comment]
Apparently, the failing tests are known, and this is not ready to merge.

File: features/activities/index.feature [hunk 1]
```diff
@@ -28,7 +28,7 @@ Feature: Activities
       | activity |
     And the project "project2" has 1 issue with the following:
       |  subject | issue2 |
-    And I am already logged in as "admin"
+    And I am admin
```
Comment (ulferts) [inline features/activities/index.feature]: "I am admin" is the slower variant of 'I am already logged in as "admin"'

I am admin should only be used where the login is tested. And for that purpose, the name is probably misguiding.

File: spec/controllers/account_controller_spec.rb [hunk 2]
```diff
@@ -28,7 +28,9 @@
 
     describe "User who is not allowed to change password can't login" do
       before do
-        post "change_password", :username => 'admin',
+        admin = User.find{|u| u.admin}
```
Comment (ulferts) [inline spec/controllers/account_controller_spec.rb]: Please use User.find_by_admin(true) here, too.

**Comment by ulferts** [comment]
Lets see what travis has to say to that.

**Pull request merged**